README.md

```diff
@@ -6,141 +16,68 @@
-- Select an issue and ask to be assigned to it. Tag @phyleria
-- After implementing your contribution, submit a pull request, ensuring it follows our contribution guidelines.
+Chimoney Community Projects are open to developers, writers and designers who want to build with, write or create around the Chimoney API. You can contribute by submitting SDKs, integrations, example apps, or technical articles showing real-world use cases. To get started, explore the [open issues](https://github.com/Chimoney/chimoney-community-projects/issues) in this repo or propose your own idea.
 
-## Submission Process
+When you're ready to make a submission, fork the repo, create a branch, make your changes, and open a pull request. Be sure to include a clear description and follow the project folder structure; all contributions go into the `submissions/` directory, with your specific contribution placed in its relevant subfolder (e.g. articles in the `articles/` folder, SDKs in the `SDKs/` folder).
 
-- For **technical writers**, please submit your PR within the `Articles` folder. *Instructions are included [here](https://github.com/Chimoney/chimoney-community-projects/tree/main/submissions/Articles)* 
-- For **other contributions**, submit your PR in the general `submissions` folder.
-  
-Please make sure to follow our formatting guidelines, and provide relevant documentation to ensure smooth and efficient reviews of your pull requests.
+You can also read the full [Contribution Guidelines](https://github.com/Chimoney/chimoney-community-projects/blob/main/CONTRIBUTING.md) here to understand how to contribute effectively.
 
-## [Contribute via Gitpod](https://www.gitpod.io/docs/introduction)
-<p align="center">
-  <a href="https://gitpod.io/#https://github.com/Chimoney/Community-projects">
-    <img src="https://gitpod.io/button/open-in-gitpod.svg" alt="Open in Gitpod">
-  </a>
-</p>
+## Chimoney Hacktoberfest
 
-## Mentorship
-New to open source? No problem! We provide mentorship to help you get started. Simply join us on [Discord](https://discord.gg/Q3peDrPG95) to connect with a mentor and begin your journey.
+Chimoney has participated in [Hacktoberfest](https://hacktoberfest.com/) every year since 2022, welcoming developers, writers, and designers to contribute to our open-source community.
 
-## License
+Each October, we create beginner-friendly **`Hacktoberfest`** issues to make it easier for first-time and seasoned contributors alike to get involved.
 
-This project is licensed under the [MIT License](https://github.com/Chimoney/chimoney-community-projects/blob/main/LICENSE).
+We also offer rewards and recognition for meaningful contributions during Hacktoberfest. To explore our past Hacktoberfest recaps, see how you can get involved, and make the most of contributing to Chimoney's open-source projects, read more here.
```

Comment:
So `read more here` should include the link to the Chimoney hacktoberfest folder once you have it all set up.